### PR TITLE
chore: refactor popup using async/await and debugger should wait for webpack completion

### PR DESCRIPTION
**Pull request by arajkumar**
This PR does code refactoring using async/await for info popup and also simplifies extension debugging by waiting for the webpack to complete.

### CODE REVIEW

**Comment by fbricon** [comment]
Can you update the screenshot from the readme, so it now shows the warning?

**Comment by arajkumar** [comment]
@fbricon , updated the opt-in screenshot.

**Comment by fbricon** [comment]
image is missing

**Comment by arajkumar** [comment]
@fbricon , looks like a bug. I could see when I open the README directly.
https://github.com/redhat-developer/vscode-commons/blob/9f1779378548add38df21e6a26da11722b61abd4/README.md

**Comment by fbricon** [comment]
My bad, I was confused by the UI. File is ok

**Comment by fbricon** [comment]
so `showWarningMessage` doesn't help at all with the popup stickiness, same 15sec display before going away. We might as well keep  `showInformationMessage` and its image

**Comment by arajkumar** [comment]
@fbricon reverted back to info popup.

File: package.json [hunk 1]
```diff
@@ -20,7 +20,9 @@
     "Other"
   ],
   "preview": true,
-  "activationEvents": [],
+  "activationEvents": [
+    "onStartupFinished"
```
Comment (fbricon) [inline package.json:24]: We're actually gonna avoid eager loading for now.
Extensions should activate when actually used

**Comment by fbricon** [comment]
Thanks @arajkumar

**Pull request merged**